packages/styled/src/index.ts

```diff
@@ -16,6 +16,7 @@ export {
   useMedia,
 } from './helper'
 export { defineThemeVariables } from './util'
+export { removeHalfLeadingCss } from './utils/typographyCss'
```

Comment:
When converting theme(o => to raw css using codemod inlining removeHalfLeadingCss itself will be too verbose.